crates/wit-parser/tests/ui/parse-fail/duplicate-value.wit.result

```diff
@@ -1,5 +1,5 @@
-`a` is defined more than once
-     --> tests/ui/parse-fail/duplicate-value.wit:5:3
+expected '(', found keyword `s32`
```

Comment:
I think this test should now be deleted given we can't declare values this way in interfaces.